server.go

```diff
@@ -82,6 +82,7 @@ type service struct {
 	server interface{} // the server for service methods
 	md     map[string]*MethodDesc
 	sd     map[string]*StreamDesc
+	meta   interface{}
```

Comment:
“meta” is fine. You know it's the metadata. What other kind of “meta” would it be?

It's also an unexported value, which means brevity should trump descriptiveness. (see the vars immediately above, where md is a map of MethodDesc and sd is a map of StreamDesc.)